### PR TITLE
Define local charts mount path as env var.

### LINKED ISSUE #1563: Add tests to deploy pre loaded modules charts
Fybrik supports working with locally mounted charts, instead of a remote repository.
See https://github.com/fybrik/fybrik/blob/1c1529b4c4bfda73e90be6abd57b130e1a32ba57/charts/fybrik/templates/manager-deployment.yaml#L142,

We need to add tests for this feature.

### PR DESCRIPTION
**Pull request by revit13**
closes #1563

This PR changes local charts mount path from being defined as constant in the code to defined as env var.
It also contains a test for the case where the charts are located in a local directory and thus should not be retrieved from the registry.

### CODE REVIEW

File: pkg/helm/helm_test.go [hunk 1]
```diff
@@ -60,7 +61,7 @@ var (
 	password      = os.Getenv("DOCKER_PASSWORD")
 	insecure, _   = strconv.ParseBool(os.Getenv("DOCKER_INSECURE"))
 	chartRef      = ChartRef(hostname, namespace, chartName, tagName)
-	impl          = NewHelmerImpl(true)
+	impl          = NewHelmerImpl("")
```
Comment (shlomitk1) [inline pkg/helm/helm_test.go:64]: What is the purpose of defining TestLocalChartsMount if the path is empty?

Comment (revit13) [reply pkg/helm/helm_test.go:64]: for this test the path is not empty: please see https://github.com/fybrik/fybrik/blob/995bd0d08d4f1490b6c96524fa1613f98ae66974/pkg/helm/helm_test.go#L142

Comment (shlomitk1) [reply pkg/helm/helm_test.go:64]: It would be cleaner to remove the global definition and use the right definition for each test.

Comment (revit13) [reply pkg/helm/helm_test.go:64]: Sure, will do that. thanks

File: pkg/helm/helm_test.go [hunk 2]
```diff
@@ -93,6 +92,7 @@ func TestHelmRegistry(t *testing.T) {
 	if err != nil {
 		t.Errorf("Unable to setup test temp charts directory: %s", err)
 	}
+	var impl = NewHelmerImpl("")
```
Comment (shlomitk1) [inline pkg/helm/helm_test.go:95]: impl := NewHelmerImpl("")

File: pkg/helm/helm_test.go [hunk 3]
```diff
@@ -139,7 +139,7 @@ func TestLocalChartsMount(t *testing.T) {
 
 	rootPath := filepath.Dir(tmpChart)
 	// remove the "charts" suffix from the file path
-	impl = NewHelmerImpl(filepath.Dir(rootPath))
+	var impl = NewHelmerImpl(filepath.Dir(rootPath))
```
Comment (shlomitk1) [inline pkg/helm/helm_test.go:142]: No need to add var, replace = with :=

File: pkg/helm/helm_test.go [hunk 4]
```diff
@@ -181,7 +181,7 @@ func TestHelmRelease(t *testing.T) {
 	}
 	var err error
 	origChart := buildTestChart()
-
+	var impl = NewHelmerImpl("")
```
Comment (shlomitk1) [inline pkg/helm/helm_test.go:184]: same comment

File: pkg/helm/helm_test.go (hunk 2 above)
Comment (revit13) [reply pkg/helm/helm_test.go]: Fixed Thanks

File: pkg/helm/helm_test.go (hunk 3 above)
Comment (revit13) [reply pkg/helm/helm_test.go]: Fixed Thanks

File: pkg/helm/helm_test.go (hunk 4 above)
Comment (revit13) [reply pkg/helm/helm_test.go]: Fixed. Thanks

**Review by shlomitk1 — APPROVED** [review]

**Review by roytman — APPROVED** [review]

**Pull request merged**